### PR TITLE
Hotfix documentation

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

Fixed:
- failing doc build due to updating requirements.txt

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: docs/README.rst [hunk 1]
```diff
@@ -15,14 +15,14 @@ Install Sphinx and extensions:
 
 .. code-block::
 
+   python -m pip install --upgrade --upgrade-strategy eager --no-cache-dir .
    cd docs
-   pip install -r requirements.txt -r ../requirements-dev.txt -r ../requirements.txt 
+   pip install -r ../requirements-dev.txt -r ../requirements.txt
```
Comment (ashwini06) [inline docs/README.rst:20]: Is this correct?

```suggestion
   pip install -r ../requirements-dev.txt 
```

**Review by ashwini06 — COMMENTED** [review]
Looks good 👍

File: docs/README.rst (hunk 1 above)
Comment (hassanfa) [reply docs/README.rst:20]: This actually should be:
```
pip install -r requirements.txt -r ../requirements-dev.txt
```

Comment (hassanfa) [reply docs/README.rst:20]: fixed.

**Review by ashwini06 — APPROVED** [review]

**Pull request merged**